aces/Loris
# [dqt] Remove old dataquery

## Brief summary of changes

The old dataquery module has been replaced with the newest dqt module.

#### Testing instructions (if applicable)

1. Checkout PR
2. make clean && make dev
3. tools/raisinbread_refresh.php
4. Make sure the sql patch was applied.
5. Visit the Data Query Tool
